### ISSUE TITLE
UI Elements are bizarrely scaled

### ISSUE THREAD

**Issue by skylord-a52** [Cockatrice/Cockatrice#4207]
<b>System Information:</b>
  
Client Version: 2.7.5 (2020-08-23)
Client Operating System: Windows 10 (10.0)
Build Architecture: 64-bit
Qt Version: 5.12.6
System Locale: en_US
Install Mode: Standard
  
_______________________________________________________________________________________

  

Cockatrice's UI is really strangely scaled, with some UI elements having very large text/images and others with normal-sized text/images. Some UI elements are covered up and hard to read as a result.
![image](https://user-images.githubusercontent.com/12107211/101435512-51f84200-38c1-11eb-98f2-3fe8fed3231c.png)

I use two monitors, with different resolutions. This sort of thing looks like what happens a lot when I change monitors while a program is open, but normally restarting the program fixes it. In this case, neither restarting nor reinstalling the program worked, despite this never being a problem before. 

**Solution:**
I eventually figured out that `right click on cockatrice.exe -> Properties -> Compatibility -> Change High DPI Settings -> Scaling performed by System (enhanced) ` fixed it, however, it'd be nice if this wasn't necessary (and hopefully even if this issue is closed, this solution will be helpful to other people).

**Complications:**
Before I reinstalled, I had another issue where the game screen was too zoomed in, and I could not see my own board/hand, making playing or spectating impossible. The other issues were fixed by the setting change, but that one was not until I reinstalled. I'm really not sure if it's related or not, and I have not been able to reproduce it.

**Comment by ebbit1q** [comment]
This is because of the high dpi screen, we are tracking this issue here: https://github.com/Cockatrice/Cockatrice/issues/1937
I'll be closing this as a duplicate.
